version

```diff
@@ -1 +1 @@
-v4.0.2
+v4.0.3
```

Comment:
Please leave this file as is, it's only changed when a new release is created.

Reply:
@max-rocket-internet done

Reply:
@max-rocket-internet done